app/components/meetings/api.js

```diff
@@ -272,6 +272,45 @@ export default class Api
     await axios.put(url, file, config);
   }
 
+  async getMeetingDocuments(options) {
+    const documents = await this.http.get(`/api/v2016/documents`, options).then(res => res.data).catch(tryCastToApiError);
+    return documents;
+  } 
+
+  async getMeetings(options) {
+    const meetings = await this.http.get(`/api/v2016/meetings`, options).then(res => res.data).catch(tryCastToApiError);
+    return meetings;
+  }
+
+  async getNotifications(options) {
+
+    const queryParameters = {
+      
+      params : {
+          q : `symbol_s (${codes.map(solr.escape).join(' or ')})`,
+          fl : "id, symbol_s,reference_s,title_t,date_dt,url_ss",
+      }
+    }
+    const notifications = await api.queryNotifications(queryParameters).then(res => res.data).catch(tryCastToApiError);
+    return notifications;
+  }
+
+  async queryNotifications({q: userQ, ...otherParams}) {
+    let q = 'schema_s:notification'
+
+    if(userQ) q = `${q} AND (${userQ})`;
+
+    const params = { ...otherParams, q };
+
+    const notifications = await this.http.get('/api/v2013/index', { params }).then(res => res.data).catch(tryCastToApiError);
+    return notifications;
+  }
```

Comment:
You can add option to cache the cache result

utilmatly you should return only the records parrt from index


```suggestion
  async queryNotifications({q: userQ, cache,  ...otherParams}) {
  
  cache = !!cache;
    let q = 'schema_s:notification'

    if(userQ) q = `${q} AND (${userQ})`;

    const params = { ...otherParams, q };

    const res = await this.http.get('/api/v2013/index', { params, cache }).then(res => res.data).catch(tryCastToApiError);
    return res;
  }
```

```javascript

    const res = await this.http.get('/api/v2013/index', { params, cache }).then(res => res.data).catch(tryCastToApiError);
    return res.response.docs;

```

Reply:
Cache included and issue with q and fl fixed.